qa/automation/ScoringPortal.Tests.Scenarios/signin_with_google_tests.feature

```diff
@@ -0,0 +1,84 @@
+
+
+Feature: sign-in via google account
+
+    """
```

Comment:
should describe functionality to test

Reply:
Do you mean to describe it in """ """  like docstring like "Checking a functionality to log-in via google account" ? I mean to which extent it should be described? Because signing in seems too obvious after all